cypress/support/fragments/exportManager/exportManagerSearchPane.js

```diff
@@ -1,4 +1,4 @@
-import { Pane, Button, TextField, MultiColumnListCell, Accordion, Checkbox, Modal } from "../../../../interactors";
+import { Pane, Button, TextField, MultiColumnListCell, Accordion, Checkbox, Modal, Section, SelectionOption } from "../../../../interactors";
```

Comment:
'Section' is defined but never used. Allowed unused vars must match /React/u

Reply:
Done